gateway_test.go

```diff
@@ -921,6 +921,9 @@ func TestListener(t *testing.T) {
 		{method: "GET", path: "/sample", code: 404},
 		{method: "GET", path: "/tyk/reload/?block=true", adminAuth: true, code: 200},
 		{method: "GET", path: "/sample", code: 200},
+		{method: "GET", path: "/samplefoo", code: 200},
+		{method: "GET", path: "/sample/", code: 200},
+		{method: "GET", path: "/sample/foo", code: 200},
```

Comment:
somewhat to mention, when it comes to standard string-values I would prefer to use already defined constants vs typing them (source of errors), like for http-verbs defined in `net/http` we have `MethodGet `

Reply:
What constant are you talking about? `"GET"`, `"/sample"`, or `200`?

All three of them are short, so I'm not sure that named constants are necessary. `net/http` itself is full of `"GET"` literals, for example. The constants there are defined for consistency more than for usefulness.

Reply:
Also worth noting that a typo in any of these literals would result in a test failure or panic, so I'm not too worried about the mistakes we could make either.

Reply:
no worries, the test code is totally fine. I meant that for http-verbs like `GET` (and yeah - response codes like `200`) we have predefined constants in `net/http` which we could use - `http.MethodGet` and `http.StatusOK`